client/scss/components/_listing.scss

```diff
@@ -472,12 +468,20 @@ table.listing {
 }
 
 // explorer specific tweaks
+.page-explorer .w-breadcrumb {
+  li[hidden] ~ li:last-child,
+  li:only-child {
+    a {
+      font-size: theme('fontSize.18');
+    }
+  }
+}
```

Comment:
This feels quite complex in CSS, it’s hard to tell what’s going on. Once we revisit those styles, I think we’ll want to try to write this with HTML classes if possible.